### ISSUE TITLE
pathProjector::Progressive does not work with time-varying right hand side

### ISSUE THREAD

**Issue by florent-lamiraux** [humanoid-path-planner/hpp-core#323]
https://github.com/humanoid-path-planner/hpp-core/blob/e0363c599865bddc7150f0a984f1f0974e43d729/src/path-projector/progressive.cc#L141
and at other places, the constraints are evaluated without refering to the path parameter.
As a consequence the path projector returns false with paths built by steeringMethod::EndEffectorTrajectory.

@jmirabel : can you confirm this statement and tell me how you overcome this issue in past applications ?

**Comment by jmirabel** [comment]
Yes, the path projector will not work with a time-varying constraint. In the end, I was using the `pathPlanner::EndEffectorTrajectory`. It has an elementary path projection method built-in. Assuming a sufficiently large number of steps, I never encountered any problems.
